plugins/woocommerce/client/blocks/assets/js/blocks/checkout/inner-blocks/checkout-actions-block/block.tsx

```diff
@@ -54,6 +55,7 @@ const Block = ( {
 
 	return (
 		<div className={ clsx( 'wc-block-checkout__actions', className ) }>
+			<CheckoutOrderSummarySlot />
```

Comment:
I understand why you had to move this here. But I hope we'll get rid of this duplicated Order Summary, because, outside of the [known issues](https://github.com/woocommerce/woocommerce/issues/58618), it also doesn't make sense to have it _hidden_ in another Inner block. If we unlock this block we'll run into the same issue. 

But it fixes the current issue so I'll approve.